cypress/fixtures/profile.json

```diff
@@ -0,0 +1,5 @@
+{
+  "id": 8739,
+  "name": "Jane",
+  "email": "jane@example.com"
+}
```

Comment:
EOF = this means that you should add one blank line to the end of the file. That's why GitHub shows this icon
![image](https://user-images.githubusercontent.com/10209180/126650199-8d0b4f07-a08b-453c-aa3d-10c2ad8d54f5.png)